src/js/utils/background.js

```diff
@@ -140,12 +140,39 @@ export const backgroundStyle = (backgroundArg, theme, textColorArg) => {
   if (background.image) {
     // allow both background color and image, in case the image doesn't fill
     return css`
+      ${!background.opacity &&
+        `
       background-image: ${background.image};
       background-repeat: ${background.repeat || 'no-repeat'};
       background-position: ${background.position || 'center center'};
-      background-size: ${background.size || 'cover'};
+      background-size: ${background.size || 'cover'};`}
       ${backgroundColor ? `background-color: ${backgroundColor};` : ''}
       ${textColor ? `color: ${textColor};` : ''}
+      ${
+        background.opacity
+          ? `
+        position: relative;
+        z-index: 0;
+        &:before {
+        content: '';
+        position: absolute;
+        background-image: ${background.image};
+        background-repeat: ${background.repeat || 'no-repeat'};
+        background-position: ${background.position || 'center center'};
+        background-size: ${background.size || 'cover'};
```

Comment:
Whether or not opacity is set all these styles are in common for the two cases, can we optimize the structure of the background a little better so there will be less code? maybe have a function for opacityStyle?

Reply:
Good thought, let me look into optimizing this.